Release notes for 8.15.4 (#16611)
elastic/logstash: docs/static/releasenotes.asciidoc

* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-15-4]]
=== Logstash 8.15.4 Release Notes

[[notable-8.15.4]]
==== Notable issues fixed

> * Fixed an issue where Logstash could not consume lines correctly when the input buffer was full while utilizing a codec with a delimiter https://github.com/elastic/logstash/pull/16482[#16482]

Reviewer: @andsel Does this line properly describe the bugfix of BufferedTokenizer?

Reviewer 2: To me seem great 👍

Reviewer 3: Can we consider this: 

"**Fixed an issue where Logstash could not consume lines correctly when a codec with a delimiter is in use and the input buffer becomes full.**"

This suggestion puts the more permanent condition first (codec with a delimiter) and the more transitional condition (buffer fills up) later.

Reviewer: It works. Applied the suggestion

> * Fixed an issue where Logstash could not consume lines correctly when the input buffer was full while utilizing a codec with a delimiter https://github.com/elastic/logstash/pull/16482[#16482]

Reviewer: ```suggestion
* Fixed an issue where Logstash could not consume lines correctly when a codec with a delimiter is in use and the input buffer becomes full https://github.com/elastic/logstash/pull/16482[#16482]
```

==== Plugins

*Cef Codec - 6.2.8*

> * [Doc] Added `raw_data_field` to docs. https://github.com/logstash-plugins/logstash-codec-cef/pull/105[#105]

Reviewer 3: ```suggestion
* [DOC] Added `raw_data_field` to docs https://github.com/logstash-plugins/logstash-codec-cef/pull/105[#105]
```

*Elastic_integration Filter - 0.1.15*

* Fixed the connection failure where SSL verification mode is disabled over SSL connection https://github.com/elastic/logstash-filter-elastic_integration/pull/165[#165]

*Geoip Filter - 7.3.1*

> * Avoided to crash pipelines when lookup a database with customised fields. https://github.com/logstash-plugins/logstash-filter-geoip/pull/225[#225]

Reviewer 3: ```suggestion
* Fixed issue causing pipelines to crash during lookup when a database has custom fields https://github.com/logstash-plugins/logstash-filter-geoip/pull/225[#225]
```

Reviewer 3: Please keep me honest that I didn't change the meaning here.
